src/Request/Data/Client/ClientData.php

```diff
@@ -175,10 +176,10 @@ public function setCompanySize(?string $companySize): ClientData
     }
 
     /**
-     * @param string $currencyCode
+     * @param CurrencyEnum $currencyCode
      * @return $this
      */
-    public function setCurrencyCode(string $currencyCode): ClientData
+    public function setCurrencyCode(CurrencyEnum $currencyCode): ClientData
```

Comment:
To avoid breaking change, deprecate the method, create a new one that is setCurrencyCodeEnum